go-controller/pkg/ovn/master.go

```diff
@@ -903,11 +907,8 @@ func (oc *DefaultNetworkController) deleteNodeEvent(node *kapi.Node) error {
 
 func (oc *DefaultNetworkController) deleteOVNNodeEvent(node *kapi.Node) error {
 	if config.HybridOverlay.Enabled {
-		if _, ok := node.Annotations[hotypes.HybridOverlayDRMAC]; ok && !util.NoHostSubnet(node) {
-			oc.deleteHybridOverlayPort(node)
-		}
-		if err := oc.removeHybridLRPolicySharedGW(node); err != nil {
-			return err
+		if err := oc.deleteHybridOverlayPort(node); err != nil {
+			return fmt.Errorf("failed to delete hybrid overlay switch port for node %s: %v", node.Name, err)
```

Comment:
I think you had a previous comment from @trozet to wrap this error with `%w` that you might have lost through the changes

Reply:
done, sorry I missed it